regexec.c

```diff
@@ -9993,7 +9993,7 @@ NULL
         default:
             PerlIO_printf(Perl_error_log, "%" UVxf " %d\n",
                           PTR2UV(scan), OP(scan));
-            Perl_croak(aTHX_ "regexp memory corruption");
+            croak("regexp memory corruption");
```

Comment:
panic

Reply:
That sounds like a good suggestion for another branch. In this branch I'm aiming not to have any behavioral changes.